### ISSUE TITLE
Reorder fields

### ISSUE THREAD

**Issue by JDziurlaj** [csg-org/esb-data-standard#8]
The fields are out of alphabetical order. Specifically

```
ApplicationRequestOtherMethod
ApplicationRequestPostmarkDate
ApplicationRequestProcessedDate
ApplicationRequestReceivedDate
ApplicationRequestStatusOtherType
ApplicationRequestStatusType
ApplicationRequestRejectionType
ApplicationRequestMethod
```

ought to be

```
ApplicationRequestMethod
ApplicationRequestOtherMethod
ApplicationRequestPostmarkDate
ApplicationRequestProcessedDate
ApplicationRequestReceivedDate
ApplicationRequestRejectionType
ApplicationRequestStatusOtherType
ApplicationRequestStatusType
```

Assuming we are using alphabetical ordering.

**Comment by jungshadow** [comment]
Alphabetical, but [with exceptions](https://github.com/votinginfoproject/vip-specification/blob/vip52/STYLEGUIDE.md#elementattribute-ordering). You bring up a different point, though. It might be worthwhile to adopt the VIP styleguide for this project rather than refer to it separately.

**Comment by JDziurlaj** [comment]
Resolved via #12
